lib/utils.js

```diff
@@ -40,6 +40,14 @@ var generateQueryString ={
     var truncateReviewsAt = query.truncateReviewsAt || '1000';
     var variationPage = query.variationPage || 'All';
     var domain = query.domain || 'webservices.amazon.com';
+
+    if(idType ==='ASIN')
```

Comment:
could be written as short ternary statement:
searchIndex = idType === 'ASIN' ? '' : '&SearchIndex=' + searchIndex